jobs/config.json

```diff
@@ -9000,7 +9000,7 @@
       "--node-tests=true",
       "--provider=gce",
       "--test_args=--nodes=1 --focus=\"\\[Serial\\]\" --skip=\"\\[Flaky\\]|\\[Benchmark\\]\"",
-      "--timeout=300m"
+      "--timeout=180m"
```

Comment:
hmm is this intentional??

Reply:
ref https://github.com/kubernetes/test-infra/pull/6046, cc @dashpole @yguo0905 this need to be in the generated config

Reply:
I though the presubmit check would catch this in https://github.com/kubernetes/test-infra/pull/6046.